src/equations/acoustic_perturbation_2d.jl

```diff
@@ -346,7 +348,8 @@ end
                                   equations)
     diss = -0.5 * λ * (u_rr - u_ll)
```

Comment:
I would prefer

```suggestion
    diss = -0.5f0 * λ * (u_rr - u_ll)
```

 in a case like this. The explicit conversion below means that we're still using `Float64` in the code involving `0.5`, which we don't want on GPUs.

Reply:
If we use `Float64` everywhere and `convert` it to something else afterward, we will still be using slow `Float64` computations on GPUs - that's exactly one of the reasons we're looking into this.
I think it would be nice to proceed as follows:
- convert `0.5 * ...` to `0.5f0 * ...`
- see whether there are similarly clear cases
- document all clear cases like `0.5` explicitly in our docs (e.g., https://trixi-framework.github.io/Trixi.jl/stable/conventions/)
- add comments to the code whenever we do something else to allow general types

Reply:
But for some other functions contain values like 0.1, it is inevitable to compute using `Float64` on GPU. What is your priority? Make it as fast as possible or keeping the code consistent with the original one. I just think that the mixed using of `Float32` and `Float64` is ugly and it will make the codebase confusing for new contributors.

Reply:
I agree that mixing types is not necessarily nice. Still, I think that we should aim at avoiding hard-coded `Float64` computations (on GPUs). Thus, using `zero(...)`, `one(...)` is fine and something like `oftype(...)` should get a comment to clarify why we write it like this.